src/module/item/SR5Item.ts

```diff
@@ -1154,6 +1154,15 @@ export class SR5Item extends Item {
         }
     }
 
+    async getLinkedActor() {    
```

Comment:
While I agree this not needing documentation for itself, I'd like to have an example use case to avoid confusion with an ownedItems parent actor:

/**
 * Retrieve the actor document linked to this item. 
 * e.g.: Contact items provide linked actors
 */